src/sentry/rules/processor.py

```diff
@@ -26,6 +26,16 @@
 SLOW_CONDITION_MATCHES = ["event_frequency"]
 
 
+def get_match_function(match_name: str) -> Callable[..., bool] | None:
```

Comment:
couldn't really find an appropriate utils file to put this in, let me know if there is one